pkg/controller/download_request_controller.go

```diff
@@ -101,7 +99,7 @@ func (c *downloadRequestController) Run(ctx context.Context, numWorkers int) err
 	var wg sync.WaitGroup
 
 	defer func() {
-		glog.Infof("Waiting for workers to finish their work")
+		c.logger.Info("Waiting for workers to finish their work")
```

Comment:
should this line come after the call to c.queue.ShutDown()?

Reply:
This should be consistent with all the other controllers. The intent is to log that we're waiting, then shut down the queue so we're actually doing so.